core/src/main/java/tech/tablesaw/columns/numbers/DoubleParser.java

```diff
@@ -42,6 +42,12 @@ public double parseDouble(String s) {
     if (isMissing(s)) {
       return DoubleColumnType.missingValueIndicator();
     }
-    return Double.parseDouble(AbstractColumnParser.remove(s, ','));
+    boolean isPercentage = false;
+    if (percentage && s.endsWith("%")) {
```

Comment:
hmm. 
So if i use the percentage() option, and the column's data doesn't end in % nothing happens. Is that what we want? What if I have a column where the numbers are percents but their is no trailing % sign? I would think that 10.4 should be converted to .104 even without a trailing percent sign, if the user has requested the transformation. 

Also, should this decision be made on a row-by-row basis? It seems like if the user says this is a percentage column we should treat the entire column as percents, otherwise a row where the percent sign is missing would not get converted. 

I guess the column might contain both 10% and .10, where both are intended to mean 10% but that kind of sloppy data is really something the user should fix before analysis, I think.

Reply:
This parser is designed to help with the apparently text columns such as 10% which are actually numeric. I guess a column transformation could be done, but it seems quite practical and I've encountered CSV files like this so the practice exists in the wild.

I don't think it's practical to think about malformed files. The user is assumed to know why they picked the percentage parser.

Converting the numeric 10.4 value to 0.104 seems less usual but I should probably add it.

Reply:
> So if i use the percentage() option, and the column's data doesn't end in % nothing happens. Is that what we want?

Yes. This flag is just telling the reader that % numbers are numbers and not a string. Without a % the normal parsing will happen.

> Also, should this decision be made on a row-by-row basis?

Yes, because there's no other mechanism. We have no per-column flags.

Reply:
> I don't think it's practical to think about malformed files. The user is assumed to know why they picked the percentage parser.

That's kinda my point. It seems like if they say the column contains percentages, we should shift all the decimal points. Or throw an exception or something consistent. 

In any case, as long as the builder documentation for the operation specifies what happens it probably doesn't matter too much which solution we choose.

Reply:
>Also, should this decision be made on a row-by-row basis?
>>Yes, because there's no other mechanism. We have no per-column flags.

yeah, but the row logic could be 

if (val ends in % or PERCENT_FLAG)

could it not?

Reply:
But they don't say a given column contains percentages. They say that if a column looks line 'nnn%' it should be treated as numeric and read as a percentage. The same way that columns that look like a date get read as date_time.